routers/web/web.go

```diff
@@ -828,6 +829,12 @@ func registerRoutes(m *web.Route) {
 						}, reqPackageAccess(perm.AccessModeWrite))
 					})
 				})
+				m.Group("/upload", func() {
+					m.Get("", user.UploadPackageChoose)
+					m.Get("/{upload_type}", user.UploadPackagePage)
+					m.Post("/generic/upload", web.Bind(forms.PackageUploadGenericForm{}), packages.UploadGenericPackagePost)
+					m.Post("/debian/upload", web.Bind(forms.PackageUploadDebianForm{}), packages.UploadDebianPackagePost)
```

Comment:
A single endpoint with a type parameter may be cleaner at the end.

Reply:
Is this possible when I need to bind different Forms for the same Endpoint?

Reply:
You could resolve the form in that method but you are right, then the individual endpoints may be better.